guides/source/caching_with_rails.md

```diff
@@ -407,15 +407,18 @@ First, add the `connection_pool` gem to your Gemfile:
 gem 'connection_pool'
 ```
 
-Next, pass the `:pool_size` and/or `:pool_timeout` options when configuring the cache store:
+Next, set `:pool` option to `true` when configuring the cache store:
 
 ```ruby
-config.cache_store = :mem_cache_store, "cache.example.com", { pool_size: 5, pool_timeout: 5 }
+config.cache_store = :mem_cache_store, "cache.example.com", pool: true
```

Comment:
I think we should really make pooling the default. Most people deploy Rails in a threaded environment now, and having a single redis connection can be a major contention point.

Can be a followup though.

Reply:
Makes sense, but I agree, it probably would be better as a followup.

Reply:
@casperisfine When enabling pooling by default, how do you think we need to make `connection_pool` a requirement? I am not sure making `connection_pool` a dependency of `active_support` is a good idea?

Reply:
@fatkodima, no we can just display an error just like if you use `mem_cache_store` and are missing `dalli`.